src/cdomains/apron/apronDomain.apron.ml

```diff
@@ -660,13 +660,13 @@ struct
   let empty_env = Environment.make [||] [||]
 
   let bot () =
-    top_env empty_env
+    bot_env empty_env
 
   let top () =
-    failwith "D2.top"
+    top_env empty_env
```

Comment:
So the conclusion is that Apron bottom and top on an empty environment are different? We should comment on it here because it's not obvious.

Maybe even adding a unit test for it, because this is an important assumption we now make. And that assumption has to be true for all of our relational domains: all the various Apron ones but also affine equalities and soon #1297.